File: 0283-move-zeroes/solution.c
void moveZeroes(int* nums, int n) {
    int l=0;
    for(int r=0;r<n;r++){
        if(nums[r]!=0){
            int temp = nums[l];
            nums[l]=nums[r];
            nums[r]=temp;
            ++l;
        }
    }
}

File: 0283-move-zeroes/solution.java
class Solution {
    public void moveZeroes(int[] nums) {
        int n=nums.length;
        int[] ans = new int[n];
        int k=0;
        for(int i=0;i<n;i++){
            ans[i]=0;
            if(nums[i]!=0){
                ans[k++]=nums[i];
            }
        }
        for(int i=0;i<n;i++){
            nums[i]=ans[i];
        }
        
    }
}
